recipe/meta.yaml

```diff
@@ -1,60 +1,188 @@
+    folder: clr
+    patches:
+      - 0001-Don-t-try-to-install-windows-files-on-linux.patch
+      - 0001-Mark-raw-strings.patch
+      - 0002-Remove-download-of-libamd64.so.5.patch
+
+  - url: https://github.com/ROCm/HIPCC/archive/refs/tags/rocm-{{ version }}.tar.gz
+    sha256: d6209b14fccdd00d7231dec4b4f962aa23914b9dde389ba961370e8ba918bde5
+    folder: hipcc
+    patches:
+      - 0001-Remove-QUIET-from-cmake-rocm-detection.patch
+      - 0002-Make-rocm-cmake-required.patch
+      - 0003-Fix-cmake-backward-compatibility.patch
 
 build:
-  number: 1
+  number: 0
   skip: True  # [not linux]
-  merge_build_host: True
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ cdt('xorg-x11-proto-devel') }}
+    - {{ cdt('libx11-devel') }}
+    - {{ cdt('mesa-libgl-devel') }}
+    - {{ cdt('mesa-libegl-devel') }}
+    - sysroot_linux-64 2.17  # [linux64]
     - cmake
     - make
-    - perl *
+    - cppheaderparser
+    - wget
   host:
-    - rocr-runtime         {{ ".".join(version.split(".")[:2]) }}
-    - rocm-device-libs     {{ ".".join(version.split(".")[:2]) }}
-    - roct-thunk-interface {{ ".".join(version.split(".")[:2]) }}
-    - rocm-comgr           {{ ".".join(version.split(".")[:2]) }}
-    - hcc                  {{ version }}
-    - clangdev 9.0.*
-    - llvmdev 9.0.*
-    - llvm
-    - perl *
-  run:
-    - {{ pin_compatible("rocr-runtime", max_pin="x.x") }}
-    - {{ pin_compatible("rocm-comgr", max_pin="x.x") }}
-    - {{ pin_compatible("roct-thunk-interface", max_pin="x.x") }}
-    - hcc                  {{ version }}
-    - rocminfo
-    - perl *
-
-test:
-  commands:
-    - $PREFIX/bin/hipcc --amdgpu-target=gfx803 --version
-    - $PREFIX/bin/hipify-clang --version
-    - $PREFIX/bin/hipconfig
+    - clangdev 17.*
+    - numactl
+    - llvmdev  17.*
+    - ocl-icd
+    - python
+    - rocm-comgr {{ major_minor_version }}.*
+    - rocr-runtime {{ major_minor_version }}.*
+    - rocm-cmake {{ major_minor_version }}.*
+    - zlib
+
+outputs:
+  - name: hip-runtime-amd
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - sysroot_linux-64 2.17  # [linux64]
+      host:
+        - numactl
+        - rocm-comgr {{ major_minor_version }}.*
+        - rocr-runtime {{ major_minor_version }}.*
+      run:
+        - libnuma  # to work around missing run_exports in numactl
+        - rocm-comgr {{ major_minor_version }}.*
+        - rocr-runtime {{ major_minor_version }}.*
+    files:
+      - lib/.hipInfo
+      - lib/cmake/hip/hip-targets*.cmake
+      - lib/cmake/hip-lang/*.cmake
+      - lib/cmake/hiprtc/*.cmake
+      - lib/libamdhip64.so*
+      - lib/libhiprtc.so*
+      - lib/libhiprtc-builtins.so*
+      - share/doc/hip/LICENSE.txt
+    test:
+      commands:
+        - test -f $PREFIX/lib/.hipInfo
+        - test -f $PREFIX/lib/libamdhip64.so
+
+  - name: hip-devel
```

Comment:
Add a run_exports to the runtime?

Reply:
Hm. Is that meaningful? I mean, the runtime needs to be a run_export for the devel package so that it gets added to the run requirements of packages that have linked against it, right? But which package would use the runtime explicitly in its host section and need it during runtime?

Or am I missing something? You certainly have more experience here...

Reply:
You already have the correct run_exports. Didn't see that.